app/Http/Controllers/Api/ProjectController.php

```diff
@@ -29,13 +33,17 @@ public function show(Request $request, Project $project){
 
     public function add_project(Request $request){
         $validator = Validator::make($request->all(), [
-            'project_address_id' => 'required|string',
             'forename' => 'required|string',
             'surname' => 'required|string',
             'project_name' => 'required|string',
             'description' => 'required|string',
             'contact_mobile_no' => 'required|string',
             'contact_email' => 'required|string',
+            'images' => 'required|array',
+            'images.*' => 'required|max:'.(config('const.dropzone_max_file_size')*1024).'|mimetypes:'.config('const.dropzone_accepted_image'),
+        ],$messages = [
+            'mimes' => 'Please insert image only',
```

Comment:
Show the supported images in the mimes after fetching from the  const.dropzone_accepted_image

Reply:
resolved